pkg/api/types.go

```diff
@@ -1410,6 +1412,55 @@ type ReplicationControllerList struct {
 	Items []ReplicationController `json:"items"`
 }
 
+// DisruptionBudget is an object to define the max disruption that can be caused to a collection of pods
+type DisruptionBudget struct {
```

Comment:
It's going to be tedious to keep a backpointer up to date.

Examples:
* Remove or change a label to orphan a pod for debugging.
* Delete a controller and orphan its pods.
* Adopt a pod under a new controller.
* Create a new service for a set of pods.

I'll try to think more about other alternatives.